weather_forecast_retrieval/data/hrrr/file_loader.py

```diff
@@ -100,28 +105,13 @@ def get_saved_data(self,
 
         self.force_zone_number = force_zone_number
 
-        if output_dir is not None:
-            self.output_dir = output_dir
-
-        if forecast_flag:
-            # TODO: Implement forecast retrieval here
-            raise NotImplementedError(
-                'Getting the forecast is not implemented yet')
-        else:
-            self.log.info('Getting saved data')
-            self.get_data(var_map)
+        if file_dir is not None:
+            self.file_dir = file_dir
 
-        metadata, dataframe = self.convert_to_dataframes(var_map)
+        self.log.info('Getting saved data')
+        self.get_data(var_map)
 
-        # manipulate data in necessary ways
-        for key in dataframe.keys():
-            dataframe[key].sort_index(axis=0, inplace=True)
-            if key == 'air_temp':
-                dataframe['air_temp'] -= 273.15
-            if key == 'cloud_factor':
-                dataframe['cloud_factor'] = 1 - dataframe['cloud_factor'] / 100
-
-        return metadata, dataframe
```

Comment:
Moved this into the `convert_to_dataframes` method. Seems a better place and consolidates any dataframe logic.